virttest/utils_libvirt/libvirt_bios.py

```diff
@@ -0,0 +1,30 @@
+"""
+Libvirt BIOS related utilities.
+
+:copyright: 2022 Red Hat Inc.
+"""
+
+import logging
+
+
+LOG = logging.getLogger('avocado.' + __name__)
+
+
+def remove_items_for_efi(vmxml):
```

Comment:
it looks like all deletion operates on  os_xml, so maybe passing os_xml looks a little natural,rather than vmxml.
And the method name could be remove_bootconfig_items_from_vmos ??

Reply:
updated